TESTS.md

```diff
@@ -292,6 +292,28 @@ test
   ,)
 ```
 
+# Record syntax
+
+Pattern matching, short
+
+```haskell
+fun Rec {alpha = beta, gamma = delta, epsilon = zeta, eta = theta, iota = kappa} = do
+  beta + delta + zeta + theta + kappa
```

Comment:
You have a point. Unfortunately this is not mentioned in Johan Tibell's style guide.
What follows is my personal opinion on spacing around braces (`{}`).

What about one-line record declarations?

I prefer
```haskell
data User = User { getName :: String }
newtype Bourne a = Bourne { getIdentity :: a }
```
to
```haskell
data User = User {getName :: String}
newtype Bourne a = Bourne {getIdentity :: a}
```
To me it makes sense to consequently also prefer `Rec { foo = 1 }` to `Rec {foo = 1}`.

What about braces around blocks? `case x of { _ -> 1 }` vs `case x of {_ -> 1}` or `do { x <- run; return (x + 1) }` vs `do {x <- run; return (x + 1)}`.

Can't we special case braces (`{}`)?